Karamelfile

```diff
@@ -8,17 +8,16 @@ dependencies:
       - ndb::mysqld
   - recipe: hops::nn
     global:
+      - hops::rm     
       - ndb::mysqld_tls
       - kagent::default
-      - hops::ndb
```

Comment:
This needs to be restored, otherwise the you'll start the namenode without the schema